src/stores/chartStore.ts

```diff
@@ -36,6 +36,7 @@ class ChartStore extends BaseStore {
 
   private widget: any;
   private shouldHandleOutsideClick = false;
+  private instrument: InstrumentModel;
```

Comment:
may be it make sense to get selected instrument directly from uiStore and not to save it into local variable?